portality/models/account.py

```diff
@@ -49,6 +49,12 @@ def pull_by_email(cls, email):
         else:
             return None
 
+    @classmethod
+    def pull_by_name(cls, name):
```

Comment:
I'm against this method, and it looks like it doesn't get used anywhere, so can be removed.